README.md

```
@@ -1,38 +1,189 @@
+</p>
+
+### Arabic Keyboard
+
+<p float="left">
+  <img src="homework-screenshots/6_arabic-keyboard.png" width="400" /> 
+</p>
+
+### Vocabulary Importer
+
+<p float="left">
+  <img src="homework-screenshots/7_vocab-importer-dark-mode.png" width="400" />
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)